iis-wf/iis-wf-transformers/src/main/resources/eu/dnetlib/iis/wf/transformers/metadataextraction/skip_extracted_without_meta/oozie_app/lib/scripts/transformer.pig

```diff
@@ -0,0 +1,24 @@
+define avro_load_document_content
+org.apache.pig.piggybank.storage.avro.AvroStorage(
+'schema', '$schema_document_content');
+
+define avro_load_document_meta
+org.apache.pig.piggybank.storage.avro.AvroStorage(
+'schema', '$schema_document_meta');
+
+define avro_store_document_content
+org.apache.pig.piggybank.storage.avro.AvroStorage(
+'index', '0',
+'schema', '$schema_document_content');
+
+documentContent = load '$input_document_content' using avro_load_document_content;
+documentMeta = load '$input_document_meta' using avro_load_document_meta;
+
+cachedDocumentId = foreach documentMeta generate id;
+cachedDocumentIdDistinct = distinct cachedDocumentId;
+
+joinedDocumentContent = join documentContent by id left, cachedDocumentIdDistinct by id;
+joinedFilteredDocumentContent = filter joinedDocumentContent by cachedDocumentIdDistinct::id is null;
```

Comment:
It seems like left outer join is more than we need.
Inner join would be enough and you've could skip filter instruction that way.